src/doc/gui_manual/WorkingWithFiles/Read_Write_Options.rst

```diff
@@ -0,0 +1,184 @@
+
+Checking this option will cause the plugin to try to guess that similarly named
+variables are the scalar components of an aggregate type such as a vector,
+tensor or array variable. The plugin will then automatically define expressions
+for these aggregate typed variables. For example, it will cause the plugin to
+combine three scalar variables with names such as ``velx``, ``vely`` and ``velz``
+into a *vector* expression ``vel`` defined as ``{velx, vely, velz}``. Note that
+this is just a convenience to free users from having to define expressions
+manally within their VisIt_ session.
+
+
+Use Material Convention
+"""""""""""""""""""""""
+
+With this option, the user can cause the plugin to recognize standard or custom
+*material conventions*. The Exodus file format does not define any specific
+standards for handling advecting and *mixing* materials. Different data
+producers have defined different conventions. A few pre-defined conventions for
+handling mixed materials from Exodus files are supported. In addition, users can
+define their own custom conventions as well. For a custom convention, the user
+must define the *namescheme* that will produce the names of the scalar variables
+holding material volume fractions. Optionally, users can specify a namescheme to
+produce the names of the scalar variables holding material-specific values for an
+associated non-material-specific variable.
+
+The
+`nameschemes <https://wci.llnl.gov/content/assets/docs/simulation/computer-codes/silo/LLNL-SM-654357.pdf?#page=226>`_
+used here are identical to those described in the
+`Silo user's manual <https://wci.llnl.gov/content/assets/docs/simulation/computer-codes/silo/LLNL-SM-654357.pdf>`_
+with one extension. The conversion specifier ``%V`` is used to denote the basename
+(non-material-specific) name of a set of scalar variables holding material
+specific values.
+
+The ALEGRA nameschemes for volume fraction and material specific variables are
+``"@%s@n?'&VOLFRC_%d&n&':'VOID_FRC':@"`` and ``"@%V_%d@n"``.
+
+The CTH nameschemes are ``"@%s@n?'&VOLM_%d&n&':'VOID_FRC':@"`` and ``"@%V_%d@n"``.
+
+Finally, in all cases it is assumed materials are identified starting from index
+one (1).  The special material id of zero (0) is used to denote void.
+
+Material Count
+++++++++++++++
+
+Ordinarily, the plugin will determine the material count from the material
+convention nameschemes. However, if it is having trouble getting the correct
+count, users can specify it manually with this option. 
+
+ffp
+~~~
+The ffp plugin can optionally use the
+`STRIPACK library <https://people.sc.fsu.edu/~jburkardt/f_src/stripack/stripack.html>`_
+to improve its behavior and performance. It will do so by loading the STRIPACK library as
+a dynamicaly loaded shared library *when* VisIt_ can find it. VisIt_ will find the
+STRIPACK library if it is available in the VisIt_ installation's top-level ``lib``
+directory (typically something like) or if the enviornment variable 
```

Comment:
Not sure if I'm misreading this somehow, but I'm getting a bit confused with the 'typically something like' part here.